### PR TITLE
Juan O SGW 663 add wallet_watchAsset RPC method

### PR DESCRIPTION
**Pull request by juanolmedo1**

### CODE REVIEW

File: stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<head>
+  <meta
+    name="description"
+    content="Adds an L0 token to the Stargazer wallet."
+  />
+</head>
+
+<intro-end />
+
+Adds an L0 token to the Stargazer wallet.
```
Comment (AlexBrandes) [inline stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md:14]: Do we support ETH/EVM networks with this RPC method also?

File: stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+`String<Boolean>` - True if the token was added successfully.
+
+##### Example
+
+```typescript title="TypeScript"
+await provider.request({
+  method: "wallet_watchAsset",
+  params: [
+    {
+      type: "L0",
```
Comment (AlexBrandes) [inline stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md:52]: What are the other "types" used for EVM networks? I'm not sure "L0" is the best option here.

File: stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md (hunk 1 above)
Comment (juanolmedo1) [reply stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md:14]: This RPC method was only added to the `constellation` provider at the moment. We'll add it to the EVM provider in the future. cc @joaoCarvalho1000

File: stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md (hunk 2 above)
Comment (juanolmedo1) [reply stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md:52]: [EIP-747](https://eips.ethereum.org/EIPS/eip-747) defines two types for EVM networks: `ERC20` and `ERC1046`. We can definitely change the `L0` type used here.

**Pull request merged**